images/openwisp_nginx/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nginx:alpine
 
-RUN apk add --update --no-cache certbot~=1.27.0-r0 openssl=1.1.1p-r0 certbot-nginx~=1.27.0-r0 && \
+RUN apk add --update --no-cache certbot~=1.27.0-r0 openssl=1.1.1q-r0 certbot-nginx~=1.27.0-r0 && \
```

Comment:
what are these changes in the dockerfiles?

Reply:
I have rebased it on the current master branch. These changes have been already merged in the master branch. I used the commits from #250 to verify that the CI passes.